llvmlite/tests/test_binding.py

```diff
@@ -1941,6 +2006,100 @@ def test_get_section_content(self):
                 self.assertEqual(s.data().hex(), issue_632_text)
 
 
+# Skip this test on Windows due to the fact that
+# the /GL cross-module optimization has the side-effect
+# of removing external symbols in this test case.
```

Comment:
Maybe adding a `DEF` file can fix this; see https://devblogs.microsoft.com/oldnewthing/20140321-00/?p=1433